packages/react-core/src/helpers/Popper/Popper.tsx

```diff
@@ -41,7 +41,7 @@ export interface PopperProps {
    */
   reference?: HTMLElement | (() => HTMLElement) | React.RefObject<any>;
   /** The popper (menu/tooltip/popover) element */
-  popper: React.ReactElement;
+  popper: any;
```

Comment:
can this type be more specific? (sorry I know I did that to poc the function method)